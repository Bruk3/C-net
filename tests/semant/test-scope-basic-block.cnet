
int sum(int x, int y, int z)
{
	y; /* this is the formal */
	{
		float y;
		char [][]z;

		y; /* this is the float y */

		for(x = 3 ; x > 2; x -= 5)
		{
			int []y;

			y; /* the int[] y */
			x; /* the x from the formals */
			z; /* the z from the block above */
		}
	}
}

int main()
{
}
