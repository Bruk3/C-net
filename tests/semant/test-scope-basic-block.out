int sum(int x,int y,int z)
{
(int : y);
{
float y = (float : 0.);
char[][] z = (char[][] : );
(float : y);
for ((int : x=(int : 3)) ; (int : (int : x) > (int : 2)) ; (int : x=(int : (int : x) - (int : 5))))
	 {
int[] y = (int[] : );
(int[] : y);
(int : x);
(char[][] : z);
}
}
return (int : 0);
}

int main()
{
return (int : 0);
}

